src/client/game.cpp

```diff
@@ -2250,7 +2250,10 @@ void Game::toggleDebug()
 			m_game_ui->m_flags.show_basic_debug = true;
 		m_game_ui->m_flags.show_profiler_graph = false;
 		draw_control->show_wireframe = true;
-		m_game_ui->showTranslatedStatusText("Wireframe shown");
+		if (driver->getDriverType() == video::EDT_OGLES2)
+			m_game_ui->showTranslatedStatusText("Wireframe shown, but not supported in OpenGL ES");
```

Comment:
I think "exposing" this detail in this way is bad.
we should either:
* skip wireframe if not supported
* just not mention this at all

you may want to wait for #15622, which makes the logic in this function more straightforward

Reply:
>you may want to wait for https://github.com/minetest/minetest/pull/15622, which makes the logic in this function more straightforward

Oh, I had not seen that it touches this. Will wait then.

> I think "exposing" this detail in this way is bad. we should either:
> 
>     * skip wireframe if not supported
> 
>     * just not mention this at all

Wouldn't this just cause confusion?
Exposing it may look ugly, but it's most user-friendly.